DB/api/test_query_db.py

```diff
@@ -84,6 +84,8 @@ def test_classify(self):
         self.retrieve_args[0] = 5
 
         self.query_options['classify'] = True
+        self.query_options['device'] = 'Panel1'
+        self.retrieve_args[7] = 'Panel1'
```

Comment:
Better to put these in order. Ex self.retrieve_args[0], self.retrieve_args[6], self.retrieve_args[7]